.github/workflows/semantic-pr.yml

```diff
@@ -0,0 +1,103 @@
+---
+name: "Semantic PR"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      # Please look up the latest version from
+      # https://github.com/amannn/action-semantic-pull-request/releases
+      - uses: amannn/action-semantic-pull-request@v3.4.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # Configure which types are allowed.
+          # Default: https://github.com/commitizen/conventional-commit-types
+          types: |
+            feat
+            fix
+            docs
+            style
+            refactor
+            perf
+            test
+            build
+            ci
+            chore
+            revert
+          # Configure which scopes are allowed.
+          scopes: |
+            aaa_server
+            agw
+            amf
+            ci
+            cloud
+            control_proxy
+            csfb
+            ctraced
+            cwg
+            directoryd
+            eap
+            eap_aka
+            eap_sim
+            enodebd
+            envoy_controller
+            eventd
+            feg
+            feg_hello
+            health
+            magmad
+            mme
+            mobilityd
+            monitord
+            nms
+            orc8r
+            pipelined
+            policydb
+            radiusd
+            redirectd
+            redis
+            s6a_proxy
+            s8_proxy
+            session_proxy
+            sessiond
+            smf
+            smsd
+            state
+            subscriberdb
+            swx_proxy
+            td-agent-bit
+            upf
+            xwf
+          # Configure that a scope must always be provided.
+          requireScope: false
+          # For work-in-progress PRs you can typically use draft pull requests
+          # from Github. However, private repositories on the free plan don't have
+          # this option and therefore this action allows you to opt-in to using the
+          # special "[WIP]" prefix to indicate this state. This will avoid the
+          # validation of the PR title and the pull request checks remain pending.
+          # Note that a second check will be reported if this is enabled.
+          wip: true
+          # When using "Squash and merge" on a PR with only one commit, GitHub
+          # will suggest using that commit message instead of the PR title for the
+          # merge commit, and it's easy to commit this by mistake. Enable this option
+          # to also validate the commit message for one commit PRs.
+          validateSingleCommit: true
```

Comment:
Oh this is nice, maybe this is a reason to stay with this gh actions workflow for the check (?)